### ISSUE TITLE
(0.15.0-beta2) Crash when LiveTv menu options are selected

### ISSUE THREAD

**Issue by MichaelRUSF** [jellyfin/jellyfin-androidtv#2281]
### Describe the bug

App crashes on .15b2 when Recorded TV, Schedule, or Series is selected from the LiveTV row. These items worked properly on the last stable build (.14.5)

### Logs

---
client: Jellyfin for Android TV
client_version: 0.15.0-beta.2
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 
```log
java.lang.NullPointerException
	at java.util.Objects.requireNonNull(Objects.java:203)
	at org.jellyfin.androidtv.ui.browsing.BrowseGridFragment.onCreate(BrowseGridFragment.java:164)
	at androidx.fragment.app.Fragment.performCreate(Fragment.java:3090)
	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:475)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:257)
	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1899)
	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1817)
	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1760)
	at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:547)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7356)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:932)

```  
***Logcat***: 
```log
--------- beginning of main
11-21 13:22:07.714 I/StartupActivity$onPermissionsGranted(26240): CurrentUser changed to 2e695138-b100-4cd4-89c0-bc17f36c8608 while waiting for startup.
11-21 13:22:07.715 D/StartupActivity(26240): Determining next activity (action=android.intent.action.MAIN, itemId=null, itemIsUserView=false)
11-21 13:22:07.749 I/org.jellyfin.sdk.api.client.KtorClient(26240): POST http://192.168.1.39:8096/Sessions/Capabilities?playableMediaTypes=Video&playableMediaTypes=Audio&supportedCommands=DisplayMessage&supportedCommands=SendString&supportedCommands=DisplayContent&supportsMediaControl=true&supportsSync=false&supportsPersistentIdentifier=true
11-21 13:22:07.809 D/Compose Focus(26240): Owner FocusChanged(true)
11-21 13:22:07.829 D/mali_winsys(26240): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, EGLBoolean) returns 0x3000
11-21 13:22:07.978 D/org.jellyfin.sdk.api.sockets.SocketInstance(26240): Credential update requested
11-21 13:22:07.978 D/org.jellyfin.sdk.api.sockets.SocketInstance(26240): Unable to update credentials: credentials did not change
11-21 13:22:08.032 D/gralloc (26240): mali_gralloc_select_format: req_format=0x00000001, usage=0x20000b00, req_base_format=0x1, alloc_format=0x8700000001, internal_format=0x8700000001
11-21 13:22:08.174 I/org.jellyfin.sdk.api.client.KtorClient(26240): GET http://192.168.1.39:8096/Playback/BitrateTest?size=4000000
11-21 13:22:08.245 D/NavigationRepositoryImpl(26240): Navigating to Fragment(fragment=class org.jellyfin.androidtv.ui.home.HomeFragment (Kotlin reflection is not available), arguments=Bundle[{}])
11-21 13:22:08.246 D/StartupActivity(26240): Opening next activity Intent { flg=0x10004000 cmp=org.jellyfin.androidtv/.ui.browsing.MainActivity }
11-21 13:22:08.327 D/gralloc (26240): mali_gralloc_select_format: req_format=0x00000001, usage=0x20000b00, req_base_format=0x1, alloc_format=0x8700000001, internal_format=0x8700000001
11-21 13:22:08.345 D/gralloc (26240): mali_gralloc_select_format: req_format=0x00000001, usage=0x20000b00, req_base_format=0x1, alloc_format=0x8700000001, internal_format=0x8700000001
11-21 13:22:08.374 D/ACRA    (26240): onActivityPaused class org.jellyfin.androidtv.ui.startup.StartupActivity
11-21 13:22:08.390 W/ActivityThread(26240): handleWindowVisibility: no activity for token android.os.BinderProxy@1dd1940
11-21 13:22:08.403 I/org.jellyfin.sdk.api.client.KtorClient(26240): GET http://192.168.1.39:8096/Shows/NextUp?userId=2e695138-b100-4cd4-89c0-bc17f36c8608&limit=10&fields=DateCreated&imageTypeLimit=1&enableTotalRecordCount=true&disableFirstEpisode=false&enableRewatching=false
11-21 13:22:08.405 I/AppThemeCallbacks(26240): Applying theme: DARK
11-21 13:22:08.406 D/ACRA    (26240): onActivityCreated class org.jellyfin.androidtv.ui.browsing.MainActivity
11-21 13:22:08.412 I/org.jellyfin.sdk.api.client.KtorClient(26240): GET http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Resume?limit=10&fields=DateCreated&mediaTypes=Video&imageTypeLimit=1&includeItemTypes=Episode&includeItemTypes=Movie&enableTotalRecordCount=true&enableImages=true&excludeActiveSessions=true
11-21 13:22:08.416 I/BackgroundService(26240): Adding BackgroundServiceFragment to activity
11-21 13:22:08.424 D/ACRA    (26240): onActivityStarted class org.jellyfin.androidtv.ui.browsing.MainActivity
11-21 13:22:08.428 D/ACRA    (26240): onActivityResumed class org.jellyfin.androidtv.ui.browsing.MainActivity
11-21 13:22:08.429 D/BackgroundServiceFragment(26240): Restoring background drawable
11-21 13:22:08.490 D/MediaManager(26240): Added event listener.  Total listeners: 1
11-21 13:22:08.498 I/org.jellyfin.sdk.api.client.KtorClient(26240): GET http://192.168.1.39:8096/LiveTv/Programs/Recommended?userId=2e695138-b100-4cd4-89c0-bc17f36c8608&limit=1&isAiring=true&imageTypeLimit=1&enableTotalRecordCount=false
11-21 13:22:08.514 I/HomeRowsFragment(26240): Updating audio queue in HomeFragment (onResume)
11-21 13:22:08.546 D/mali_winsys(26240): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, EGLBoolean) returns 0x3000
11-21 13:22:08.562 D/gralloc (26240): mali_gralloc_select_format: req_format=0x00000001, usage=0x20000b00, req_base_format=0x1, alloc_format=0x8700000001, internal_format=0x8700000001
11-21 13:22:08.591 D/gralloc (26240): mali_gralloc_select_format: req_format=0x00000001, usage=0x20000b00, req_base_format=0x1, alloc_format=0x8700000001, internal_format=0x8700000001
11-21 13:22:08.599 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Views?format=json
11-21 13:22:08.600 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Shows/NextUp?Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&UserId=2e695138-b100-4cd4-89c0-bc17f36c8608&Limit=50&ImageTypeLimit=1&format=json
11-21 13:22:08.603 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items?SortBy=DatePlayed&EnableTotalRecordCount=false&ExcludeLocationTypes=Virtual&MediaTypes=Video&ImageTypeLimit=1&recursive=true&Filters=IsResumable&Fields=PrimaryImageAspectRatio%2COverview%2CItemCounts%2CDisplayPreferencesId%2CChildCount&sortOrder=Descending&Limit=50&format=json
11-21 13:22:08.605 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/LiveTv/Programs/Recommended?Fields=Overview%2CPrimaryImageAspectRatio%2CChannelInfo%2CChildCount&UserId=2e695138-b100-4cd4-89c0-bc17f36c8608&EnableTotalRecordCount=false&IsAiring=true&Limit=20&ImageTypeLimit=1&format=json
11-21 13:22:08.606 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Shows/NextUp?Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&UserId=2e695138-b100-4cd4-89c0-bc17f36c8608&Limit=50&ImageTypeLimit=1&format=json
11-21 13:22:08.611 I/org.jellyfin.sdk.api.client.KtorClient(26240): GET http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Views?includeHidden=false
11-21 13:22:08.735 I/org.jellyfin.sdk.api.client.KtorClient(26240): GET http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Views?includeHidden=false
11-21 13:22:08.743 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=d49a1e03-df26-2eef-307a-b2ec0b67fa4e&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:08.744 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=f137a2dd-21bb-c1b9-9aa5-c0f6bf02a805&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:08.744 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=f53fc094-64c1-5245-f537-3088e52a7f1a&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:08.745 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=9fcf63d9-3232-c8f3-246c-83c6611ed78d&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:08.746 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=a656b907-eb3a-7353-2e40-e44b968d0225&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:08.747 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=bdf38141-c3a3-66eb-1a2a-8240d2e65e68&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:08.748 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=d7e58e58-ad49-209c-5917-1d6a3ecd6952&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:08.751 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=7e64e319-657a-9516-ec78-490da03edccb&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:08.756 D/jellyfin-apiclient(26240): Adding request to queue: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=79a2726d-3c50-e769-a8af-1e4184e4fccf&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:08.933 D/gralloc (26240): mali_gralloc_select_format: req_format=0x00000001, usage=0x20000b00, req_base_format=0x1, alloc_format=0x8700000001, internal_format=0x8700000001
11-21 13:22:08.935 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Views?format=json
11-21 13:22:08.947 D/ItemRowAdapterHelperKt(26240): Creating items from 0 existing and 11 new, adapter size is 0
11-21 13:22:08.969 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/LiveTv/Programs/Recommended?Fields=Overview%2CPrimaryImageAspectRatio%2CChannelInfo%2CChildCount&UserId=2e695138-b100-4cd4-89c0-bc17f36c8608&EnableTotalRecordCount=false&IsAiring=true&Limit=20&ImageTypeLimit=1&format=json
11-21 13:22:08.985 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items?SortBy=DatePlayed&EnableTotalRecordCount=false&ExcludeLocationTypes=Virtual&MediaTypes=Video&ImageTypeLimit=1&recursive=true&Filters=IsResumable&Fields=PrimaryImageAspectRatio%2COverview%2CItemCounts%2CDisplayPreferencesId%2CChildCount&sortOrder=Descending&Limit=50&format=json
11-21 13:22:08.990 D/ItemRowAdapterHelperKt(26240): Creating items from 0 existing and 4 new, adapter size is 0
11-21 13:22:08.996 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Shows/NextUp?Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&UserId=2e695138-b100-4cd4-89c0-bc17f36c8608&Limit=50&ImageTypeLimit=1&format=json
11-21 13:22:09.133 I/lyfin.androidt(26240): Background young concurrent copying GC freed 103109(3981KB) AllocSpace objects, 23(732KB) LOS objects, 5% free, 19MB/20MB, paused 433us total 315.275ms
11-21 13:22:09.535 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Shows/NextUp?Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&UserId=2e695138-b100-4cd4-89c0-bc17f36c8608&Limit=50&ImageTypeLimit=1&format=json
11-21 13:22:09.642 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=d49a1e03-df26-2eef-307a-b2ec0b67fa4e&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:09.652 I/OpenGLRenderer(26240): Davey! duration=879ms; Flags=0, IntendedVsync=11519756922532, Vsync=11519990255856, OldestInputEvent=9223372036854775807, NewestInputEvent=0, HandleInputStart=11520002408586, AnimationStart=11520002435586, PerformTraversalsStart=11520002912419, DrawStart=11520378356169, SyncQueued=11520402750086, SyncStart=11520403369002, IssueDrawCommandsStart=11520519428169, SwapBuffers=11520632120252, FrameCompleted=11520636702002, DequeueBufferDuration=3568000, QueueBufferDuration=361000, 
11-21 13:22:09.666 D/ItemRowAdapterHelperKt(26240): Creating items from 0 existing and 28 new, adapter size is 0
11-21 13:22:09.696 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=9fcf63d9-3232-c8f3-246c-83c6611ed78d&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:09.700 D/ItemRowAdapterHelperKt(26240): Creating items from 0 existing and 12 new, adapter size is 0
11-21 13:22:09.720 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=f53fc094-64c1-5245-f537-3088e52a7f1a&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:09.736 D/ItemRowAdapterHelperKt(26240): Creating items from 0 existing and 31 new, adapter size is 0
11-21 13:22:09.771 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=f137a2dd-21bb-c1b9-9aa5-c0f6bf02a805&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:09.785 D/ItemRowAdapterHelperKt(26240): Creating items from 0 existing and 50 new, adapter size is 0
11-21 13:22:09.838 D/ACRA    (26240): onActivityStopped class org.jellyfin.androidtv.ui.startup.StartupActivity
11-21 13:22:09.855 D/ACRA    (26240): onActivityDestroyed class org.jellyfin.androidtv.ui.startup.StartupActivity
11-21 13:22:09.872 D/Compose Focus(26240): Owner FocusChanged(false)
11-21 13:22:09.872 D/Compose Focus(26240): Owner FocusChanged(true)
11-21 13:22:09.872 D/Compose Focus(26240): Owner FocusChanged(false)
11-21 13:22:09.879 I/lyfin.androidt(26240): Background concurrent copying GC freed 122767(6214KB) AllocSpace objects, 27(1928KB) LOS objects, 49% free, 16MB/32MB, paused 5.022ms total 734.355ms
11-21 13:22:09.896 W/libEGL  (26240): EGLNativeWindowType 0xdcd0b6a8 disconnect failed
11-21 13:22:09.903 I/Choreographer(26240): Skipped 52 frames!  The application may be doing too much work on its main thread.
11-21 13:22:09.984 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=bdf38141-c3a3-66eb-1a2a-8240d2e65e68&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:09.988 D/ItemRowAdapterHelperKt(26240): Creating items from 0 existing and 9 new, adapter size is 0
11-21 13:22:09.991 I/OpenGLRenderer(26240): Davey! duration=968ms; Flags=0, IntendedVsync=11520007138452, Vsync=11520873805084, OldestInputEvent=9223372036854775807, NewestInputEvent=0, HandleInputStart=11520887878002, AnimationStart=11520887900502, PerformTraversalsStart=11520899049002, DrawStart=11520960328669, SyncQueued=11520962412419, SyncStart=11520962886335, IssueDrawCommandsStart=11520967955585, SwapBuffers=11520974663085, FrameCompleted=11520976497002, DequeueBufferDuration=258000, QueueBufferDuration=364000, 
11-21 13:22:09.995 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=a656b907-eb3a-7353-2e40-e44b968d0225&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:09.998 D/ItemRowAdapterHelperKt(26240): Creating items from 0 existing and 27 new, adapter size is 0
11-21 13:22:10.017 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=d7e58e58-ad49-209c-5917-1d6a3ecd6952&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:10.026 D/ItemRowAdapterHelperKt(26240): Creating items from 0 existing and 50 new, adapter size is 0
11-21 13:22:10.049 I/WM-WorkerWrapper(26240): Worker result SUCCESS for Work [ id=b05144b5-0d51-488b-898d-64c75fd5e79e, tags={ org.jellyfin.androidtv.integration.LeanbackChannelWorker } ]
11-21 13:22:10.072 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=7e64e319-657a-9516-ec78-490da03edccb&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:10.079 D/ItemRowAdapterHelperKt(26240): Creating items from 0 existing and 35 new, adapter size is 0
11-21 13:22:10.130 I/jellyfin-apiclient(26240): Response received from: http://192.168.1.39:8096/Users/2e695138-b100-4cd4-89c0-bc17f36c8608/Items/Latest?ParentId=79a2726d-3c50-e769-a8af-1e4184e4fccf&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
11-21 13:22:10.131 D/ItemRowAdapterHelperKt(26240): Creating items from 0 existing and 2 new, adapter size is 0
11-21 13:22:10.909 I/AutoBitrate(26240): Auto bitrate set to: 11560693
11-21 13:22:17.778 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(26240): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
11-21 13:22:17.781 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession(26240): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"f3e7dcce153f4dbaabf83d5ccad34328"}
11-21 13:22:18.678 D/NavigationRepositoryImpl(26240): Navigating to Fragment(fragment=class org.jellyfin.androidtv.ui.browsing.BrowseGridFragment (Kotlin reflection is not available), arguments=Bundle[{folder={"Name":"Series Recordings","Id":"29683dcf-3208-4ec0-b389-c23285f71375","Type":"Folder","CollectionType":"SeriesTimers"}}])
11-21 13:22:18.702 D/AndroidRuntime(26240): Shutting down VM
--------- beginning of crash
11-21 13:22:18.703 E/AndroidRuntime(26240): FATAL EXCEPTION: main
11-21 13:22:18.703 E/AndroidRuntime(26240): Process: org.jellyfin.androidtv, PID: 26240
11-21 13:22:18.703 E/AndroidRuntime(26240): java.lang.NullPointerException
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at java.util.Objects.requireNonNull(Objects.java:203)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at org.jellyfin.androidtv.ui.browsing.BrowseGridFragment.onCreate(BrowseGridFragment.java:164)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at androidx.fragment.app.Fragment.performCreate(Fragment.java:3090)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:475)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:257)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1899)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1817)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1760)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:547)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at android.os.Handler.handleCallback(Handler.java:883)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at android.os.Handler.dispatchMessage(Handler.java:100)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at android.os.Looper.loop(Looper.java:214)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at android.app.ActivityThread.main(ActivityThread.java:7356)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at java.lang.reflect.Method.invoke(Native Method)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
11-21 13:22:18.703 E/AndroidRuntime(26240): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:932)
11-21 13:22:18.703 E/ACRA    (26240): ACRA caught a NullPointerException for org.jellyfin.androidtv
11-21 13:22:18.703 E/ACRA    (26240): java.lang.NullPointerException
11-21 13:22:18.703 E/ACRA    (26240): 	at java.util.Objects.requireNonNull(Objects.java:203)
11-21 13:22:18.703 E/ACRA    (26240): 	at org.jellyfin.androidtv.ui.browsing.BrowseGridFragment.onCreate(BrowseGridFragment.java:164)
11-21 13:22:18.703 E/ACRA    (26240): 	at androidx.fragment.app.Fragment.performCreate(Fragment.java:3090)
11-21 13:22:18.703 E/ACRA    (26240): 	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:475)
11-21 13:22:18.703 E/ACRA    (26240): 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:257)
11-21 13:22:18.703 E/ACRA    (26240): 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1899)
11-21 13:22:18.703 E/ACRA    (26240): 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1817)
11-21 13:22:18.703 E/ACRA    (26240): 	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1760)
11-21 13:22:18.703 E/ACRA    (26240): 	at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:547)
11-21 13:22:18.703 E/ACRA    (26240): 	at android.os.Handler.handleCallback(Handler.java:883)
11-21 13:22:18.703 E/ACRA    (26240): 	at android.os.Handler.dispatchMessage(Handler.java:100)
11-21 13:22:18.703 E/ACRA    (26240): 	at android.os.Looper.loop(Looper.java:214)
11-21 13:22:18.703 E/ACRA    (26240): 	at android.app.ActivityThread.main(ActivityThread.java:7356)
11-21 13:22:18.703 E/ACRA    (26240): 	at java.lang.reflect.Method.invoke(Native Method)
11-21 13:22:18.703 E/ACRA    (26240): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
11-21 13:22:18.703 E/ACRA    (26240): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:932)
11-21 13:22:18.703 D/ACRA    (26240): Building report
11-21 13:22:18.704 D/ACRA    (26240): Starting collectors with priority FIRST
11-21 13:22:18.705 D/ACRA    (26240): Calling collector org.acra.collector.DropBoxCollector
11-21 13:22:18.705 D/ACRA    (26240): Collector org.acra.collector.DropBoxCollector completed
11-21 13:22:18.705 D/ACRA    (26240): Calling collector org.acra.collector.LogCatCollector
11-21 13:22:18.705 D/ACRA    (26240): Calling collector org.acra.collector.StacktraceCollector
11-21 13:22:18.707 D/ACRA    (26240): Collector org.acra.collector.StacktraceCollector completed
11-21 13:22:18.711 D/ACRA    (26240): Retrieving logcat output (buffer:default)...
11-21 13:22:18.732 E/libc    (27189): Access denied finding property "persist.vendor.system.svp.dbg.loglevel"

```  

### App information

***App version***: `0.15.0-beta.2` (`150002`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"mt5895","BOOTLOADER":"unknown","BRAND":"Sony","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"BRAVIA_VH2","DISPLAY":"BRAVIA_VH2_M_UC-user 10 QTG3.200305.006.S184 659421 release-keys","FINGERPRINT":"Sony\/BRAVIA_VH2_M_UC\/BRAVIA_VH2:10\/QTG3.200305.006.S184\/659421:user\/release-keys","HARDWARE":"mt5895","HOST":"BuildHost671","ID":"QTG3.200305.006.S184","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"Sony","MODEL":"BRAVIA 4K VH2","PERMISSIONS_REVIEW_REQUIRED":true,"PRODUCT":"BRAVIA_VH2_M_UC","RADIO":"unknown","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"release-keys","TIME":1657537479000,"TYPE":"user","UNKNOWN":"unknown","USER":"root","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"","CODENAME":"REL","FIRST_SDK_INT":29,"INCREMENTAL":"659421","PREVIEW_SDK_FINGERPRINT":"REL","PREVIEW_SDK_INT":0,"RELEASE":"10","RESOURCES_SDK_INT":29,"SDK":"29","SDK_INT":29,"SECURITY_PATCH":"2022-05-01"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":150002,"VERSION_NAME":"0.15.0-beta.2"}
```  

### Device information

***Android version***: `10`  
***Device brand***: `Sony`  
***Device product***: `BRAVIA_VH2_M_UC`  
***Device model***: `BRAVIA 4K VH2`  

### Crash information

***Start time***: `2022-11-21T13:14:45.602-05:00`  
***Crash time***: `2022-11-21T13:22:18.756-05:00`  

### Application version

0.15. beta2

### Where did you install the app from?

Sideloaded APK

### Device information

BRAVIA 4K VH2

### Android version

AndroidTV 10

### Jellyfin server version

10.8.7

**Comment by nielsvanvelzen** [comment]
This _should_ be fixed in the next beta, but since I don't have Live TV configured that well I couldn't test it properly. So let us know if something is still wrong after the next beta.

**Comment by MichaelRUSF** [comment]
Using dev build #2856 two of the three submenu items now work correctly (Recorded Tv & Schedule). The submenu "Series" still leads to a crash.
[upload_org.jellyfin.androidtv.debug_0.0.0-dev.1_20221126024839_031c839317774ea48bfae8afe8cc672b.log](https://github.com/jellyfin/jellyfin-androidtv/files/10095562/upload_org.jellyfin.androidtv.debug_0.0.0-dev.1_20221126024839_031c839317774ea48bfae8afe8cc672b.log)

**Comment by nielsvanvelzen** [comment]
It should now be fixed with #2295 (build: https://github.com/jellyfin/jellyfin-androidtv/suites/9523533958/artifacts/452403295)

**Comment by MichaelRUSF** [comment]
Just tested out with build 2868 and  Live TV series leads to a crash. 
 
[upload_org.jellyfin.androidtv.debug_0.0.0-dev.1_20221129162726_c7aba4619d1f40d7bb9f2787a2e5ed22.log](https://github.com/jellyfin/jellyfin-androidtv/files/10115115/upload_org.jellyfin.androidtv.debug_0.0.0-dev.1_20221129162726_c7aba4619d1f40d7bb9f2787a2e5ed22.log)

**Comment by nielsvanvelzen** [comment]
Thanks for testing, should be fixed with #2301
